akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -98,7 +106,7 @@ object ActorPath {
  * when comparing actor paths.
  */
 @SerialVersionUID(1L)
-sealed trait ActorPath extends Comparable[ActorPath] with Serializable {
+sealed abstract class ActorPath extends Comparable[ActorPath] with Serializable {
```

Comment:
We don't have mima enabled on `master`. 
I think we're currently not passing MiMa between 2.3 and 2.4 - @patriknw did experiment towards this but we didn't resolve all incompatibilities AFAIR